openapi/core-data.yaml

```diff
@@ -351,6 +351,15 @@ components:
         minimum: 0
         default: 0
       description: "The number of items to skip before starting to collect the result set."
+    readingOffsetParam:
+      in: query
+      name: offset
+      required: false
+      schema:
+        type: integer
+        minimum: 0
```

Comment:
If API allows offset to be -1, should we specify the minimum to be -1 instead of 0 here?

Reply:
You're right, updated.